lib/RouteStore.js

```diff
@@ -23,6 +25,7 @@ var RouteStore = createStore({
         this.currentUrl = null;
         this.currentRoute = null;
         this.currentNavigate = null;
+        this.currentNavigateError = null;
```

Comment:
I know that this is a WIP, but I was just wondering if `currentNavigateError` needs to be `dehydrate`'d and `rehydrate`'d so it will work on the server too?